zee/config/config.ron

```diff
@@ -1,11 +1,24 @@
 (
-    // Which theme to use
+    ////////////////////////////////////////////////////////////////////////////
+    //                                                                        //
+    // GENERAL SETTINGS                                                       //
+    //                                                                        //
+    ////////////////////////////////////////////////////////////////////////////
```

Comment:
I don't think I am a big fan of ASCII art boxes
 - in my experience it makes it harder to edit unless you have good editor support
 - it doesn't improve readability, personally I find it decreases it
   - it is "busier", there are just more characters on the screen
   - as with syntax highlighting the comments already have a different colour, they stick out as is
 
But it's obviously a style question, it's reasonable to disagree. My preference would be to stick with the previous style.

Reply:
Thank you for your feedback!

I usually place those comment banners in source files when there are major sections with different tasks.  In this case, I considered the modes a section on their own on the one side and the further, "general" settings on the other one.

It is no problem to remove them, it was considered as visual clarification for the users that they can rely on the settings' sorting due to this categorisation.  I will revert the comment banners and create a discussion for the comment style.  :wrench:

Reply:
:wrench: :heavy_check_mark: